index.js

```diff
@@ -67,6 +68,108 @@ function toggleColor(element) {
+   
+   // likes variable from line 24 is in global scope
+   // approach 1 - using likes from DOM
+   let currLikes = parseInt(likes.textContent)
+   likes.textContent = `{currLikes + 1} Likes`
+
+   // approach 2 - using likes from traveler objects
+   // likes.textContent = `${traveler.likes++} Likes`
+})
+
 /***** Deliverable 3 *****/
+
+// const form = document.querySelector('form#new-animal-sighting-form')
+
+// form.addEventListener('submit', function (event) {
+//     event.preventDefault() // ALWAYS!
+//     console.log('Form submitted!!!')
+
+//     // GET THE USER INPUT FROM THE FORM
+//     console.dir(event.target)
+
+//     const inputFieldOne = event.target[0].value
+//     const inputFieldTwo = event.target[1].value
+//     const inputFieldThree = event.target[2].value
+//     const inputFieldFour = event.target[3].value
+
+//     const newAnimalSightingObject = {
+//         id: traveler.animalSightings[traveler.animalSightings.length - 1].id + 1,
+//         travelerId: traveler.animalSightings[traveler.animalSightings.length - 1].id + 1,
+//         species: inputFieldOne,
+//         photo: inputFieldTwo,
+//         link: inputFieldThree,
+//         description: inputFieldFour,
+//     }
+
+//     traveler.animalSightings.push(newAnimalSightingObject)
+
+//     renderAnimalSightingPost(newAnimalSightingObject)
+//     form.reset()
+// })
+
+const newSightingForm = document.querySelector('form#new-animal-sighting-form')
+
+newSightingForm.addEventListener('submit', function (event) {
+    event.preventDefault()
+
+    // debugger
+
+    const species = event.target[0].value
+    const video = event.target[1].value
+    const photo = event.target[2].value
+    const description = event.target[3].value
+    
+    // another option #1
+    // const species = event.target.querySelector('input[name="species")').value
+    // const video = event.target.querySelector('input[name="link")').value
+    // const photo = event.target.querySelector('input[name="photo")').value
+
+    // another option #2 - using the id or name attribute on the element
+    // you can use dot notation if the value of the attribute doesn't have any special characters like a dash. Otherwise, use bracket notation.
+    // const species = e.target["species"].value;
+    // const videoLink = e.target["animal-link"].value;
+    // const photo = e.target["animal-photo"].value;
+    // const description = e.target["sighting-description"].value;
+
+    // const species = e.target.species.value; // ex using dot notation
+
+    const lastIndex = traveler.animalSightings.length - 1
+    const newSightingObject = {
+        id: traveler.animalSightings[lastIndex].id + 1, // won't need to do this when we incorporate our own real database.
+        travelerId: 1,
+        species: species,
+        photo: photo,
+        link: video,
+        description: description
+    }
+    renderAnimalSightingPost(newSightingObject)
+
```

Comment:
Here, we could include:
`traveler.animalSightings.push(newSightingObject)`
It is a good idea to include this line because of the way we're creating custom IDs. If we want future animal sightings to have a unique ID, then we have to push all new animal sighting objects into the array.